client/components/time-since/index.tsx

```diff
@@ -10,7 +9,7 @@ interface TimeSinceProps {
 
 function TimeSince( { className, date, dateFormat = 'll' }: TimeSinceProps ) {
 	const moment = useLocalizedMoment();
-	const fullDate = useMemo( () => moment( date ).format( 'llll' ), [ moment, date ] );
```

Comment:
non-blocking, just for information. was the useMemo the cause of the issue?

Reply:
Not `useMemo` per se, but the fact that the `moment` instance is the same, despite the change in locale.